src/utils/mime.ts

```diff
@@ -139,4 +90,6 @@ const baseMimes: Record<string, BaseMime> = {
   '3g2': 'video/3gpp2',
   gltf: 'model/gltf+json',
   glb: 'model/gltf-binary',
-}
+} as const
+
+const baseMimes: Record<string, string> = _baseMimes
```

Comment:
Should this be `Record<string, BaseMime>`?

Reply:
thanks!